<launch>
  <!-- Location of the map you want to store -->
  <arg name="map_address" value="$(find map_store)/maps/sample.yaml" />
  <!-- Name of map you want to put into the database -->
  <arg name="map_name" value="sample" />

  <param name="warehouse_host" value="localhost" />
  <param name="warehouse_port" value="27017" />

  <node pkg="map_store" type="map_saver" name="map_saver" />
  <node pkg="map_store" type="map_manager" name="map_manager" output="screen" />
  <node pkg="map_server" type="map_server" name="map_server" args="$(arg map_address)" />
  <node pkg="map_store" type="add_map.py" name="add_map" args="'$(arg map_name)'" required="true"
    output="screen" />

</launch>